MetaData.json

```diff
@@ -1,6 +1,6 @@
 {
   "Name": "CohortDiagnosticsModule",
-  "Version": "0.0.2",
+  "Version": "0.0.8",
```

Comment:
This version should be made to represent the next sequence in the release in this repository. This version number was simply my attempt to make smaller releases during my testing.